app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,24 @@ def show
     @user = User.find(params[:user_id])
     @comments = @post.comments
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+    @post.author_id = Current.user.id
+    if @post.save
+      redirect_to "/users/#{Current.user.id}/posts"
+    else
+      render :new
+    end
+  end
+
+  private
```

Comment:
- [ ] After creating the db and inserting some commands in rails c to create users and posts in the browser I can see this there it can't find the user id because of the name atrribute so please fix the column name problem and I think it will work properly 💯✨🔥